spec/models/issue_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Issue do
+
+  describe '.sync_with_github' do
+
+      it "deletes issues that are no longer open", :vcr  do
+        issue = FactoryBot.create(:issue)
+
+        Issue.sync_with_github
+        expect(Issue.all).to be_empty
+      end
+
+      it "updates issues that are open", :vcr do
+        user_bot = FactoryBot.create(
+          :user,
+          name: "ombu-bot",
+          id: 1
+          )
```

Comment:
This is the indentation I was referring. it seems the last ")" is not indented correcty
```ruby
it "updates issues that are open", :vcr do
  user_bot = FactoryBot.create(
    :user,
    name: "ombu-bot",
    id: 1
  )
end
```